src/components/panel/panel.js

```diff
@@ -923,9 +1043,23 @@ MdPanelRef.prototype.open = function() {
   return this._$q(function(resolve, reject) {
     var done = self._done(resolve, self);
     var show = self._simpleBind(self.show, self);
+    var setOpened = function(response) {
```

Comment:
@ErinCoughlan I thought that the response was necessary in order to continue the promise stream?